apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "version": "1.0.0-*",
```

Comment:
Not yet! Let's start with 0.1.0, then just before the first release I'll make it 1.0.0-alpha01
We don't want to accidentally go GA :)